plugins/destination/kafka/docs/_configuration.md

```diff
@@ -11,17 +11,19 @@ spec:
   version: "VERSION_DESTINATION_KAFKA"
   write_mode: "append"
   spec:
-    brokers: ["<broker-host>:<broker-port>"]
-    format: "csv" # options: parquet, json, csv
+    # required - list of brokers to connect to
+    brokers: ["${KAFKA_BROKER}"]
```

Comment:
Having an environment variable will make the UI experience good for a single broker, but what if we have more than one broker? Maybe we shouldn't use an environment variable here.

Reply:
Is the broker a secret? If not we can drop the env variable and let the user decide

Reply:
No, so I've removed it as suggested.